### ISSUE TITLE
No way to customize geometry of vncservice

### ISSUE THREAD

**Issue by doug-w** [binhex/arch-basegui#1]
This is more of a feature request but it would be nice before calling into vncserver that we can set some environment variables for the the-geometry parameter.  Pycharm in 1024x768 is 'challenging'

**Comment by binhex** [comment]
novnc should be running at whatever resolution your screen is set to, so when i access pycharm on my screen set to 1920x1080 resolution i can see pycharm is set to match this, and as i resize the window so the resolution changes for the remote screen running pycharm, are you perhaps using a vnc client as opposed to novnc?.

**Comment by doug-w** [comment]
Yes, I see that if I use the provided URL for novnc (resize=remote) then it does resize correctly.  

I looked into it more before replying to this.  Apparently it's a feature of tigervnc server that NoVNC uses behinds the scenes.  And only the tigervnc viewer supports it.  Once I switched to that it works perfectly as I resize the window.  It may be worth a line in a README about it.

Thanks for all your work.  I've also put a lot of time into novnc apps and really love seeing it shown off.
